DBNostalgia/NoTransactionBehavior.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DBNostalgia
+{
+    public class NoTransactionBehavior : ITransactionBehavior
```

Comment:
Temporary solution for the transactional vs non transactional problem.
This is the first iteration of this.